cvat/apps/engine/models.py

```diff
@@ -316,15 +316,23 @@ class Image(models.Model):
     class Meta:
         default_permissions = ()
 
-class Project(models.Model):
+class TimestampedModel(models.Model):
+    created_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now=True)
```

Comment:
1. Why aren't we going to update base classes for several models in other applications (that have date fields)?
1. Will it be better to separate this class into 2 with `created_date` and `updated_date` + `touch`? Otherwise, now we have some models that are not inherited from `TimestampedModel` but have the same custom `created_date` field.

Reply:
> Why aren't we going to update base classes for several models in other applications (that have date fields)?

I think I just forgot to look in other apps. Thanks for reminding me; updated.

> Will it be better to separate this class into 2 with `created_date` and `updated_date` + `touch`?

I guess in principle it would be better, but I think this refactoring would go beyond the scope of what I'm trying to do in this PR. If in the future I have to add some shared functionality involving only `created_date`, I'll consider something like this.